Lib/uuid.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             if len(bytes) != 16:
                 raise ValueError('bytes is not a 16-char string')
             assert isinstance(bytes, bytes_), repr(bytes)
-            int = int_.from_bytes(bytes, byteorder='big')
+            int = int_.from_bytes(bytes)  # big_endian
```

Comment:
```suggestion
            int = int_.from_bytes(bytes)  # big endian
```

Reply:
`big_endian` can be improved, yes. But it seems to me the adjective form is spelled with a hyphen (`big-endian`). CPython's got 97 examples of that spelling already versus only 33 "[Bb]ig endian" with space.

```suggestion
            int = int_.from_bytes(bytes)  # big-endian
```

Reply:
In fact, wait... What's the point of applying the default if you have to add a comment saying what that default is? In this case either we should trust the default to be obvious, or just not use it and be explicit instead, right?